code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -282,6 +311,15 @@
 	else
 		. += "You could improvise a shoulder sling from some cabling..."
 
+	if (reinforced)
+		. += "The barrel has been reinforced for use with high-power ammunition."
+	else if (barrel_stress == 0)
+		. += "The barrel is in pristine condition."
+	else if (barrel_stress <= 25)
+		. += "The barrel seems to be warped mildly..."
+	else
+		. += "The barrel is cracked, and the warping is more severe!"
```

Comment:
`The barrel is warped and cracked` would look better for the final message imo. 

Additionally these thresholds don't match up with the warning messages very well right now: 
Thresholds should be `<10` and `<20` rather than `== 0` and `<= 25`.

Reply:
Updated to have stuff line up. Barrel becomes warped at 10 stress (when message occurs), and cracked at 25 stress now.

After 2 shots, it's warped (20% misfire). After 5 shots, it's cracked (35% misfire).